lib/sidekiq/serializers.rb

```diff
@@ -0,0 +1,46 @@
+require "sidekiq/serializers/basic"
+
+module Sidekiq
+  module Serializers
+    def self.serializer
+      serializers[Sidekiq.default_configuration[:serialize_as]] ||
+        raise(ArgumentError, "Invalid serializer: #{Sidekiq.default_configuration[:serialize_as]}")
+    end
+
+    def self.serializers
+      @serializers ||= {
+        basic: Sidekiq::Serializers::Basic
```

Comment:
Why not use instances here, e.g. `S::Serializers::Basic.new`?

Reply:
I don't currently have a need to store state on that object.

Reply:
Put another way: this pattern gets interesting to me when you have something to pass to `new`

Reply:
I suppose we could do something similar to the middleware chain where we allow the user to pass arguments to the serializers.